GenderPayGap.Database/GpgDatabaseContextFactory.cs

```diff
@@ -14,7 +14,8 @@ public GpgDatabaseContext CreateDbContext(string[] args)
             var optionsBuilder = new DbContextOptionsBuilder<GpgDatabaseContext>();
 
             //Setup the SQL server with automatic retry on failure
-            optionsBuilder.UseSqlServer(GpgDatabaseContext.ConnectionString, options => options.EnableRetryOnFailure());
+//            optionsBuilder.UseSqlServer(GpgDatabaseContext.ConnectionString, options => options.EnableRetryOnFailure());
+            optionsBuilder.UseNpgsql(@"Server=127.0.0.1;Port=5432;Database=gpg_database;User Id=gpg_user;Password=local_gpg_database;", options => options.EnableRetryOnFailure());

```

Comment:
This probably doesn't need to be hard-coded here if we have the value in `GpgDatabaseContext.ConnectionString`

Reply:
All sorted now